modules/timedrift/timedrift.go

```diff
@@ -64,6 +65,13 @@ type ntpstats struct {
 	Reachable bool      `json:"reachable"`
 }
 
+var NtpBackupPool = []string{
+	`time.nist.gov`,
```

Comment:
Is there any reason we are treating these as raw string literals vs using a standard quote?

Not a big deal, just wondering.